statsmodels/base/l1_cvxopt.py

```diff
@@ -169,9 +169,9 @@ def _get_G(k_params):
     """
     The linear inequality constraint matrix.
     """
-    I = np.eye(k_params)
```

Comment:
This one is sill.  I has bad but A and B are ok? I has clear meaning to anyone with linear algebra knowledge, and I_k isn't an improvement.

Reply:
Oh, wrong rule,  But I is still clear, at least with any decent fixed width font.